### PR TITLE
Add com.github.Johnn3y.Forklift

### PR DESCRIPTION
**Pull request by Johnn3y**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.github.Johnn3y.Forklift

File: com.github.Johnn3y.Forklift.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
+  "id" : "com.github.Johnn3y.Forklift",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.34",
+  "runtime-commit" : "4729b7b5c9fa77deabb2c14e39ea77f7c2ea4f7c1d25c5136c8665a0d350d003",
```
Comment (bilelmoussaoui) [inline com.github.Johnn3y.Forklift.json]:
```suggestion
```

File: com.github.Johnn3y.Forklift.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  "sdk" : "org.gnome.Sdk",
+  "sdk-commit" : "fa0a5022e5da88bec837c2d1f2ff02d9be690263fa0e36e9da70657092904923",
```
Comment (bilelmoussaoui) [inline com.github.Johnn3y.Forklift.json]:
```suggestion
```

File: com.github.Johnn3y.Forklift.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  "cleanup" : [
```
Comment (bilelmoussaoui) [inline com.github.Johnn3y.Forklift.json]: Nitpick: move this after finish-args.

**Comment by Johnn3y** [comment]
bot, build com.github.Johnn3y.Forklift

File: com.github.Johnn3y.Forklift.json [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-devices",
+        "--enable-gnutls",
+        "--enable-libmp3lame",
+        "--enable-libvorbis"
+      ],
+      "sources" : [
+        {
+          "url" : "https://ffmpeg.org/releases/ffmpeg-4.0.2.tar.xz",
```
Comment (barthalion) [inline com.github.Johnn3y.Forklift.json]: 4.0.4 is out. (Or 4.2.1 if you don't need that specific release branch.)

File: com.github.Johnn3y.Forklift.json [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "Forklift",
+      "buildsystem" : "meson",
+      "sources" : [
+        {
+          "url" : "https://github.com/Johnn3y/Forklift",
```
Comment (barthalion) [inline com.github.Johnn3y.Forklift.json]: If it's a stable release, you should create a tag in your repository and then re-use it here. (Not a blocker, just a good practice.)

**Comment by Johnn3y** [comment]
bot, build com.github.Johnn3y.Forklift

**Comment by Johnn3y** [comment]
bot, build com.github.Johnn3y.Forklift

**Comment by nedrichards** [comment]
@Johnn3y I get a missing icon in the app for the Download button and the search icon in the add by text field popover.

Also, as a non compulsory tip, it'd be good if there was a 'blank slate' experience for when you open it for the first time that gave an indication of what to do, I was staring at a blank window for a few seconds trying to orient myself.

**Comment by Johnn3y** [comment]
bot, build com.github.Johnn3y.Forklift

**Comment by Johnn3y** [comment]
@nedrichards 
> @Johnn3y I get a missing icon in the app for the Download button and the search icon in the add by text field popover.

I fixed the icon issue.
> Also, as a non compulsory tip, it'd be good if there was a 'blank slate' experience for when you open it for the first time that gave an indication of what to do, I was staring at a blank window for a few seconds trying to orient myself.

I' m working on that and will add it soon. Thanks.

**Comment by nedrichards** [comment]
I get an error trying to download a video:
```
Traceback (most recent call last):
  File "/app/share/Forklift/Forklift/applicationwindow.py", line 186, in on_deleteselectedbutton_clicked
    for l in lists:
NameError: name 'lists' is not defined
Traceback (most recent call last):
  File "/app/share/Forklift/Forklift/applicationwindow.py", line 186, in on_deleteselectedbutton_clicked
    for l in lists:
NameError: name 'lists' is not defined
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/app/share/Forklift/Forklift/items.py", line 176, in run
    with cd(self.path), youtube_dl.YoutubeDL(self.ydl_opts) as ydl:
  File "/app/lib/python3.7/site-packages/youtube_dl-2019.9.28-py3.7.egg/youtube_dl/YoutubeDL.py", line 428, in __init__
    pp = pp_class(self, **compat_kwargs(pp_def))
TypeError: __init__() got an unexpected keyword argument 'format'
```

Also I have issues with the icons in the Download status box. More generally, is this app designed around only working with a specific theme? It looks like there are some spacing and alignment issues with the standard Adwaita, you may want to lock it to a specific theme and then bundle that in the manifest as well - there's some examples of other apps on Flathub (mostly from elementary) that do that if you need a hand.

**Comment by Johnn3y** [comment]
bot, build com.github.Johnn3y.Forklift

**Pull request closed without merging**